ts/indicators/ichimoku-kinko-hyo.src.ts

```diff
@@ -0,0 +1,1024 @@
+/* *
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+import H from '../parts/Globals.js';
+
+/* eslint-disable @typescript-eslint/interface-name-prefix */
+
+/**
+ * Internal types
+ * @private
+ */
+declare global {
+    namespace Highcharts {
+
+        class IKHIndicator
+            extends SMAIndicator {
+            public data: Array<IKHIndicatorPoint>;
+            public graphCollection: Array<string>;
+            public graphsenkouSpan: (SVGElement | undefined)
+            public ikhMap: Dictionary<Array<IKHIndicatorPoint>>;
+            public nextPoints: Array<IKHIndicatorPoint>;
+            public options: IKHIndicatorOptions;
+            public pointArrayMap: Array<keyof IKHIndicatorPoint>;
+            public pointValKey: string;
+            public pointClass: typeof IKHIndicatorPoint;
+            public points: Array<IKHIndicatorPoint>;
+            public drawGraph(): void;
+            public init(): void;
+            public toYData(point: Point): Array<number>;
+            public translate(): void;
+
+        }
+
+        interface IKHIndicatorDrawSenkouSpanObject {
+            indicator: IKHIndicator;
+            points: Array<IKHIndicatorPoint>;
+            nextPoints: Array<IKHIndicatorPoint>;
+            color?: ColorType;
+            options: IKHIndicatorOptions;
+            gap: IKHIndicatorGapExtensionObject;
+            graph: (SVGElement|undefined);
+        }
+
+        interface IKHIndicatorSenkouSpanObject {
```

Comment:
I guess you renamed the wrong interface yesterday. This one needs to end on `Options` (or `OptionsObject`) as it becomes part of the options tree in `IKHIndicatorOptions`.

Reply:
There you go. That is probably the culprit yes. Trying again with `IKHIndicatorSenkouSpanOptions`